implementations/rust/ockam/ockam_app/src/enroll/menu_items.rs

```diff
@@ -18,7 +18,7 @@ pub struct EnrollActions {
 impl EnrollActions {
     pub fn new() -> EnrollActions {
         let enroll = CustomMenuItem::new(ENROLL_MENU_ID, "Enroll...").accelerator("cmd+e");
-        let reset = CustomMenuItem::new(RESET_MENU_ID, "Reset...").accelerator("cmd+r");
+        let reset = CustomMenuItem::new(RESET_MENU_ID, "Reset").accelerator("cmd+r");// Updated label to "Reset"
```

Comment:
This change os already merged, please remove it from this commit.
You may need to rebase to our latest develop branch.